category_encoders/target_encoder.py

```diff
@@ -76,15 +76,22 @@ class TargetEncoder(util.BaseEncoder, util.SupervisedTransformerMixin):
     memory usage: 51.5 KB
     None
 
-    >>> X = ['N', 'N', 'NE', 'NE', 'NE', 'SE', 'SE', 'S', 'S', 'S', 'S', 'W', 'W', 'W', 'W', 'W']
-    >>> hierarchical_map = {'Compass': {'N': ('N', 'NE'), 'S': ('S', 'SE'), 'W': 'W'}}
-    >>> y = [1, 0, 1, 1, 1, 0, 0, 1, 0, 1, 0, 1, 0, 0, 0, 1]
-    >>> enc = TargetEncoder(hierarchy=hierarchical_map).fit(X, y)
-    >>> hierarchy_dataset = enc.transform(X)
-    >>> print(hierarchy_dataset[0].values)
-    [0.5        0.5        0.94039854 0.94039854 0.94039854 0.13447071
-    0.13447071 0.5        0.5        0.5        0.5        0.40179862
-    0.40179862 0.40179862 0.40179862 0.40179862]
+    >>> X, y = load_compass()
```

Comment:
maybe add the corresponding import statement to make it a self contained example
``` >>> from category_encoders.datasets import load_compass``